### DISCUSSION TITLE
Tero Subtitler 1.0.0.1

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#97] [Announcements]
Use it at your own risk!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.0.1'>Tero Subtitler 1.0.0.1</a>.</em>

**Comment by GitGianluc** [comment]
**Warning for Apple Silicon Mac users**: this build contains a bug #98 which causes the app to crash on startup. _Do not update_ and wait for a new upcoming release ([Tero Subtitler 1.0.0.2](https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.0.2))!
